txpipe/twopoint_fourier.py

```diff
@@ -222,44 +226,57 @@ def load_maps(self):
             for g2 in g2_maps:
                 w = np.where(g2!=healpy.UNSEEN)
                 g2[w]*=-1
-                
-                
-        # Load HEALPix systematics maps for clustering deprojection
+
+
+        # Load HEALPix systematics maps
         deproject_syst_clustering = self.config['deproject_syst_clustering']
-        if deproject_syst_clustering:  
+        if deproject_syst_clustering:
             print('Deprojecting systematics maps for number counts')
             n_systmaps = 0
             s_maps = []
             systmaps_clustering_dir = self.config['systmaps_clustering_dir']
             systmaps_path = pathlib.Path(systmaps_clustering_dir)
             for systmap in systmaps_path.iterdir():
-                systmap_file = str(systmap)
-                print('Reading clustering systematics map file:', systmap_file)
                 try:
                     if systmap.is_file():
-                        syst_map = healpy.read_map(systmap_file, verbose=False)
-                    else:
-                        warnings.warn(f'Map from {systmap} is missing. Skipping...')
+                        if pathlib.Path(systmap).suffix != ".fits":
```

Comment:
I think that this would break if the map is compressed? Maybe add `or pathlib.Path(systmap).suffix != ".fits.gz"`

Reply:
There are probably big savings using `fits.gz` for HSC

Reply:
@fjaviersanchez Done. In fact, I would default to `fits.gz` maps.